xl_auth/utils.py

```diff
@@ -11,3 +13,11 @@ def flash_errors(form, category='warning'):
     for field, errors in form.errors.items():
         for error in errors:
             flash('{0} - {1}'.format(getattr(form, field).label.text, error), category)
+
+
+def get_gravatar_url(user_email, size):
+    """Get gravatar url from user email."""
+    clean_email = str(user_email).lower().encode()
+    hash = hashlib.md5(clean_email)
```

Comment:
Shadows built-in name `hash`. And I think this should be a method on the `User` model instead of a stand-alone utility function. That way you can always call `current_user.get_gravatar_url(64)` without any fuzz.

Something like this:
```python
    def get_gravatar_url(self, size=32):
        """Get Gravatar URL."""
        hashed_email = hashlib.md5(str(self.email).lower()).hexdigest()
        return 'https://www.gravatar.com/avatar/{}?d=mm&s={}'.format(hashed_email, size)
```